### ISSUE TITLE
cannot determine user id for 'ntp', does the user exist on this system?

### ISSUE THREAD

**Issue by randyrue** [sous-chefs/ntp#247]
Looks like for Ubuntu 24.04 LTS server the ntp service runs as the user "ntpsec" instead of the previous "ntp"

`Running handlers:
[2024-05-02T15:03:04+00:00] ERROR: Running exception handlers
Running handlers complete
[2024-05-02T15:03:04+00:00] ERROR: Exception handlers complete
Infra Phase failed. 0 resources updated in 06 seconds
[2024-05-02T15:03:04+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
[2024-05-02T15:03:04+00:00] FATAL: ---------------------------------------------------------------------------------------
[2024-05-02T15:03:04+00:00] FATAL: PLEASE PROVIDE THE CONTENTS OF THE stacktrace.out FILE (above) IF YOU FILE A BUG REPORT
[2024-05-02T15:03:04+00:00] FATAL: ---------------------------------------------------------------------------------------
[2024-05-02T15:03:04+00:00] FATAL: Chef::Exceptions::UserIDNotFound: directory[/var/lib/ntp] (ntp::default line 71) had an error: Chef::Exceptions::UserIDNotFound: cannot determine user id for 'ntp', does the user exist on this system?`

**Comment by rurku** [comment]
Indeed, looks like not only the user changed but also some of the paths
My temporary workaround is setting these attributes
```
"ntp": {
  "var_owner": "ntpsec",
  "var_group": "ntpsec",
  "conffile": "/etc/ntpsec/ntp.conf",
  "varlibdir": "/var/lib/ntpsec",
  "statsdir": "/var/log/ntpsec/"
},
```

**Comment by damacus** [comment]
Closing this because the cookbook was rewritten in v7.0.0 to manage distro-packaged ntpsec explicitly on Debian 12+/Ubuntu 22.04+, including the ntpsec user/group and platform-specific paths such as /etc/ntpsec/ntp.conf, /var/lib/ntpsec, and /var/log/ntpsec. The current resource no longer uses the old Ubuntu 24.04 defaults that caused this failure.
